src/main/java/stsjorbsmod/cards/cull/Clarify.java

```diff
@@ -0,0 +1,85 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.RetainCardPower;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+import stsjorbsmod.patches.SelfExertField;
+import stsjorbsmod.powers.ClarifyPower;
+
+public class Clarify extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Clarify.class);
+
+    private static final CardRarity RARITY = CardRarity.RARE;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.POWER;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 0;
+    private static final int CARDS_RETAINED = 1;
+    private static final int UPGRADE_PLUS_CARDS_RETAINED = 1;
+
+    private boolean retainApplied = false;
+
+
+    public Clarify() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        selfRetain = true;
+        baseMagicNumber = magicNumber = CARDS_RETAINED;
+        SelfExertField.selfExert.set(this, true);
+    }
+
+    @Override
+    public void triggerWhenDrawn() {
```

Comment:
I feel a little uneasy about the "apply RetainCardPower when drawn and remove it when leaving hand" pattern:
* I think it's confusing for Amnesia to be able to remove it
* I think there are some edge case ways in which it can leave your hand that don't have hooks (eg, stuff that moves cards to the top of your deck)

I think it might be better to use `triggerOnEndOfTurnForPlayingCard` to queue a `RetainCardsPolitelyAction`; it doesn't cooperate quite as well with multiple copies of Clarify (it'll give 2 separate dialogs), but it doesn't outright break in any cases I'm thinking of